csharp/ql/integration-tests/windows-only/standalone_dependencies/Assemblies.expected

```diff
@@ -163,55 +161,7 @@
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/System.Xml.XPath.dll |
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/System.Xml.XmlDocument.dll |
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/System.Xml.XmlSerializer.dll |
+| /microsoft.netcore.app.ref/7.0.2/ref/net7.0/WindowsBase.dll |
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/mscorlib.dll |
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/netstandard.dll |
-| /microsoft.windowsdesktop.app.ref/7.0.2/ref/net7.0/Accessibility.dll |
-| /microsoft.windowsdesktop.app.ref/7.0.2/ref/net7.0/Microsoft.VisualBasic.Forms.dll |
```

Comment:
Is it expected that all the `microsoft.windowsdesktop.app.ref` references are gone?

Reply:
That is a good question actually. Not sure exactly how the assets file looks like on windows. Could very much be the case, that they were removed just because we haven't added these to frameworks exception list (and just include everything when we face a dependency with this name). I will experiment a bit.